src/malloc_wrapper.cc

```diff
@@ -46,7 +46,7 @@ void gnu_backtrace()
 	}
 
 	for (j = 0; j < nptrs; j++)
-		printf("%s\n", strings[j]);
+		printf("callback: %s\n", strings[j]);
 
```

Comment:
[cppcoreguidelines-avoid-c-arrays]: do not declare C-style arrays, use std::array<> instead